### PR TITLE
Populate UI with locations

### PR DESCRIPTION
**Pull request by sayers24**
Issue #46 
UI side of #52 for #45

### CODE REVIEW

**Comment by cliffckerr** [comment]
I got excited about merging this but see it's still WIP :)

File: covasim/webapp/cova_app.py [hunk 1]
```diff
@@ -105,6 +105,18 @@ def get_version():
     return output
 
 
+@app.register_RPC()
+def get_preset_options():
+    ''' Get the list of options for the location select '''
+    # TODO import locations from datasets/translators.py that @@gwincr11 is working on
```
Comment (gwincr11) [inline covasim/webapp/cova_app.py:111]: My work was merged. You should be able to do 
```
from covasim.datasets.data_loader import available_countries
available_countries()
```

File: covasim/webapp/index.html [hunk 2]
```diff
@@ -116,6 +116,18 @@ <h4 class="my-2">Campaign Builder</h4>
             {{sim_length.tip}}
           </b-tooltip>
         </div>
+        <div class="form-group">
+          <label for="country">Demographic:</label>
```
Comment (sayers24) [inline covasim/webapp/index.html:120]: @cliffckerr, not sure if we should move this to the population parameters or if it is important enough to expose in the general options

Comment (ckerr-IDM) [reply covasim/webapp/index.html:120]: @sayers24 There should just be a "location" drop-down that sets everything automatically

Comment (ckerr-IDM) [reply covasim/webapp/index.html:120]: FYI, I'm working on the other PRs that will impact this, and see also https://github.com/InstituteforDiseaseModeling/covasim/pull/171 for more merge hell :)

**Comment by cliffckerr** [comment]
@sayers24 is this still draft?

**Comment by sayers24** [comment]
@cliffckerr, I just need to address merge conflicts after #171 went in. I can work on that now.

**Comment by ckerr-IDM** [comment]
@sayers24 sweet!!

File: covasim/webapp/index.html (hunk 2 above)
Comment (sayers24) [reply covasim/webapp/index.html]: Would we want to call out that the location drop down is more than just a preset? The model uses this to take age distribution into account, correct? We could add a little more descriptive text to the tooltip.

**Comment by cliffckerr** [comment]
Thanks @sayers24 , great to have this in there :)

**Pull request merged**